src/components/totals/EntityGoalProgress/index.js

```diff
@@ -68,11 +70,17 @@ module.exports = React.createClass({
   },
 
   onSuccess: function(res) {
+    console.log(res)
+
     this.setState({
       isLoading: false,
       total: res.total_amount_cents.sum ? res.total_amount_cents.sum + this.props.offset : 0,
       goal: res.goal
     });
+
+    if (typeof this.props.onLoad === 'function') {
```

Comment:
We thought this would be worth keeping, just in case of the (hopefully) unlikely situation where someone writes something that isn't a function as the `onLoad` property.